src/main/java/oo/lang/bool/FalseClass.java

```diff
@@ -0,0 +1,11 @@
+package oo.lang.bool;
+
+public class FalseClass extends Bool {
+    public static final FalseClass False = new FalseClass();
+
```

Comment:
We can create as many `False` values as we want. But it supposed to be a singleton class. Forbid all `Bool` instances to be manually created.

Reply:
it IS supposed